README.md

```diff
@@ -45,6 +45,8 @@ Feel free to submit a [pull request](https://github.com/EddieHubCommunity/awesom
 
 4. Create a [Pull Request](https://github.com/EddieHubCommunity/awesome-github-profiles/pulls) to add the changes to this repository. Please change the title to `fix: add <your-username>` and in the description add `resolve #issueNumber`
 
+## Check out our awesome profiles [Here](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/profiles.md)
```

Comment:
```suggestion
## Check out our awesome profiles [here](https://eddiehubcommunity.github.io/awesome-github-profiles/)
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
Hey @schmelto can you please tell me what I need to fix

Reply:
ok I've got It now

Reply:
Hey @schmelto the typo error has been fixed now